### PR TITLE
Revoke permission from user onedrive

### PR DESCRIPTION
**Pull request by npsptechusa**
https://github.com/MicrosoftDocs/OfficeDocs-O365ITPro/issues/115

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix - any updates on the content edit review on this PR? Thanks.

**Comment by mypil** [comment]
@kenwith - would you be able to merge the content for this PR? 
This PR already contains the copy/edit review done by @nenonix. Thank you.

**Comment by dianef77** [comment]
@dianef77  figure out how to approve. fix portal.office.com

**Comment by mypil** [comment]
@dianef77 - Do you have any updates on this?

Thank you.

**Comment by mypil** [comment]
@dianef77 @twernermsft - Can you please assist to check, approve and merge the content of this PR?

Thank you.

cc: @scotvidican @kenwith

**Comment by mypil** [comment]
@twernermsft @cmcatee-MSFT - I would like to follow up. Can you please assist to check, approve and merge the content of this PR based on issue #115?

Thank you.

cc: @scotvidican

**Comment by cmcatee-MSFT** [comment]
@nenonix, @mypil Please remove the image from the hyperlinksto the admin center. References to the admin center should now be styled as "In the admin center, go to the **Users** > **Active users** page." The page they expect to land on should be hyperlinked using the FWLinks listed in the wiki. In this case, it's https://go.microsoft.com/fwlink/p/?linkid=834822
Please see the wiki for more details about how to write steps with deep links into the admin center. Under no circumstances should the URL point to portal.office.com, or to another topic about the admin center. Always link to the page in the admin center. If a FWLink doesn't exist for the page you need to link to, reach out to me and I will get one created for you. Thanks!

**Comment by mypil** [comment]
@npsptechusa - Please assist to make the necessary changes to this PR as recommended by @cmcatee-MSFT. Thanks.

**Comment by nenonix** [comment]
@mypil @npsptechusa -have made the changes- please merge https://github.com/npsptechusa/OfficeDocs-O365ITPro/pull/2

File: Office365-Admin/add-users/get-access-to-and-back-up-a-former-user-s-data.md [hunk 1]
```diff
@@ -30,37 +30,59 @@ description: "Learn how to preserve an employee's files and emails when the pers
...
 1. Sign in to Office 365 with your work or school account. 
     
-2. Go to the [Office 365 admin center](../admin-overview/about-the-admin-center.md).
+2. Go to the [![Office 365 admin center.](../media/e00ba917-c3fb-4173-b344-43eb5c7eeb15.png)](https://portal.office.com/adminportal/home).
     
-3. Go to Active users and select the user.
+3. Go to **Active Users** and select the user.
     
-4. Expand **OneDrive Settings** in the user details pane, and then click **Access files**.
+4. Expand **OneDrive Settings** in the user details pane, and select **Access files**.
```
Comment (cmcatee-MSFT) [inline Office365-Admin/add-users/get-access-to-and-back-up-a-former-user-s-data.md]: Change to:
`4. In the user details pane, choose **OneDrive**, then choose **Create link to files**.`

File: Office365-Admin/add-users/get-access-to-and-back-up-a-former-user-s-data.md [hunk 2]
```diff
@@ -30,37 +30,59 @@ description: "Learn how to preserve an employee's files and emails when the pers
...
     > You can move up to 500 MB of files and folders at a time.
-    > When you use Move to with documents that have version history, only the latest version is moved. To move earlier versions, you need to restore and move each one. 
+    > When you use **Move to** with documents that have version history, only the latest version is moved. To move earlier versions, you need to restore and move each one. 
+    
+## Revoke admin access to a user’s OneDrive site
+
+
+As global admin you have access to the content in a user’s OneDrive site, but you may want to remove your access to a user’s documents. By default, the OneDrive Site Collection Administrator is the owner of the OneDrive account. The following steps describe how to remove a Site Collection Admin on a user’s OneDrive site.
+
+1. Sign in to the [![Office 365 admin center.](../media/e00ba917-c3fb-4173-b344-43eb5c7eeb15.png)](https://portal.office.com/adminportal/home) as a global or SharePoint admin.
```
Comment (cmcatee-MSFT) [inline Office365-Admin/add-users/get-access-to-and-back-up-a-former-user-s-data.md]: Change to:
`1. Sign in to the <a href="https://go.microsoft.com/fwlink/p/?linkid=837890" target="_blank">admin center</a> as a global or SharePoint admin.`

**Comment by nenonix** [comment]
@cmcatee-MSFT I am creating a new PR based on my commit which will replace this one

**Comment by nenonix** [comment]
@cmcatee-MSFT @mypil New PR https://github.com/MicrosoftDocs/OfficeDocs-O365ITPro/pull/540 - this one can be closed

**Pull request closed without merging**